dbus_next/aio/message_bus.py

```diff
@@ -231,7 +259,16 @@ def send(self, msg: Message):
             self._buffered_messages.append(msg)
             return
 
-        asyncio.ensure_future(self._loop.sock_sendall(self._sock, msg._marshall()))
+        buf = msg._marshall()
+
+        async def _send():
+            ancdata = [(socket.SOL_SOCKET, socket.SCM_RIGHTS, array.array("i", msg.unix_fds))] \
+                if msg.unix_fds else None
+
+            await self.sock_sendmsg(self._sock, buf[:1], ancdata=ancdata)
```

Comment:
Same here, send the whole fixed-size header first, then the rest.

Reply:
And done